composer.json

```diff
@@ -24,6 +24,7 @@
     },
     "require-dev": {
         "phpunit/phpunit": "^6.4",
+        "symfony/framework-bundle": ">= 2.8.18",
```

Comment:
this should be in the require section instead of ^2.8 use ^2.8.18

Reply:
Look at the commit dd91b32.

I did it that way first. Then I put it in the development dependencies because it is only relevant for test execution (the famous PHPUnit namespace change).

I can drop this commit.

Reply:
ah, ok. well, symfony 2.8 is antique. i think its ok to just force 2.8.18 in general. when people update the bundle, they hopefully also update symfony itself within the 2.8 version.

Reply:
I reverted the commit.